exposee-ui/src/main.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App' 
+import { UserProvider } from './UserContext'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
+    <UserProvider>
+    <App /> 
```

Comment:
I'm surprised eslint didn't indent the App element

Reply:
i didn't run it because it messed up the first time